delivery_carrier_label_ups/tests/test_delivery_carrier_label_ups.py

```diff
@@ -99,3 +99,15 @@ def test_labels(self):
                 ),
             )
             return super().test_labels()
+
```

Comment:
`delivery_carrier_label_ups/tests/test_delivery_carrier_label_ups.py:104:5: E303 too many blank lines (2)`

Reply:
thanks, I'll have to rework some things here anyways, as I realized yesterday that it's nonsense to handle the pickup when the picking is done. This needs to happen earlier, and I'm not yet sure if I'll add some custom stuff here or extend base...label to allow a general mechanism for pickups.